changelogs/2024-03-20_nested-keys-within-preview.mdx

```diff
@@ -0,0 +1,25 @@
+---
+_schema: default
+title: Nested keys within preview
+type: minor
+---
+This release added nested keys to the card `preview` key.
+
+It also addressed several issues, including those affecting DAMs, copying links from Google Docs, Hugo `hardWraps`, and conflicting updates.
+
+### Features & Improvements:
+
+* You can now use nested keys within `preview`.
+
+### Fixes:
+
+* Fixed an issue where branched sites would not copy the state of the "upload locked" flag, allowing you to upload images to your git repository rather than your DAM.
+* Fixed an issue where CloudCannon failed to respect the "extra prefix" option when searching and uploading to larger DAMS.
+* Fixed an issue where you could not see non-image files in a Cloudinary DAM via the CloudCannon browser.
+* Fixed an issue where copying a link from Google Docs would add a `<u>` tag.
+* Fixed an issue where quotation marks would be replaced by `&quot;`.
+* Fixed an issue where Hugo `hardWraps` was false by default. The `hardWraps` key will now match your Hugo config file.
```

Comment:
The hardWraps option is still false by default, I believe. It's possibly slightly more accurate to say something like "CloudCannon now respects the `hardWraps` option for Hugo, by omitting <br> tags when inserting newlines in the Content Editor`. That's a bit of a mouthful, so I'll leave that to your discretion!